src/components/select/bl-select.css

```diff
@@ -0,0 +1,221 @@
+:host {
+  --bl-select-padding-vertical: var(--bl-size-2xs);
```

Comment:
To apply safer approach that will prevent developers to style many components unintentionally, we should create a class (ex. `bl-select`) and define variables under this class. To hear more about this, you can look at [this](https://github.com/Trendyol/baklava/discussions/164) discussion.

Reply:
I think there should be a way to customize a component intentionally. I moved these styles under a class but in my opinion z-index property should stay under :host because developers might need to change it.